src/ontology/cl-edit.owl

```diff
@@ -30031,20 +30031,20 @@ EquivalentClasses(obo:CL_4023125 ObjectIntersectionOf(obo:CL_4023123 ObjectSomeV
 
 # Class: obo:CL_4023127 (arcuate nucleus of hypothalamus KNDy neuron)
 
-AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:30042117"^^xsd:string) obo:IAO_0000115 obo:CL_4023127 "a KDNy neuron that is located in the arcuate nucleus of the hypothalamus.")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:30042117"^^xsd:string) obo:IAO_0000115 obo:CL_4023127 "a KNDy neuron that is located in the arcuate nucleus of the hypothalamus."^^xsd:string)
 AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4023127 <https://orcid.org/0000-0001-7258-9596>)
 AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4023127 <https://orcid.org/0000-0002-0819-0473>)
 AnnotationAssertion(rdfs:label obo:CL_4023127 "arcuate nucleus of hypothalamus KNDy neuron")
 EquivalentClasses(obo:CL_4023127 ObjectIntersectionOf(obo:CL_4023125 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0001932)))
 
-# Class: obo:CL_4023128 (rostral periventricular region of the third ventricle KDNy neuron)
+# Class: obo:CL_4023128 (rostral periventricular region of the third ventricle KNDy neuron)
```

Comment:
Consider adding KDNy as an exact_synonym, or making a comment reflecting that in some papers the term KDNy has been used due typos.

Reply:
Thanks for reviewing, @aleixpuigb. As discussed offline, will defer adding a comment or synonym since:
- KDNy is likely a typo
- Several terms have this abbreviation in the label as a description and want to avoid setting a precedent of adding a comment / syn to all future terms that may include the abbreviation